### ISSUE TITLE
Custom card_light_color looks strange with light theme

### ISSUE THREAD

**Issue by CM000n** [UI-Lovelace-Minimalist/UI#139]
**Is your enhancement request related to a problem? Please describe.**
Hi @basbruss I now also use your light cards and find them great! Unfortunately I have noticed that they look a bit strange in combination with the light theme:

With your card:
<img width="674" alt="Screenshot 2021-12-23 104258" src="https://user-images.githubusercontent.com/7945681/147228374-18d68d19-9df5-4f58-a1cc-6e08afb4298d.png">

With the integrated light card:
![43545435](https://user-images.githubusercontent.com/7945681/147229560-fb10502b-9967-4d65-b116-c2d8624a1dc2.PNG)

**Describe the solution you'd like**
In my opinion, the background of lights that are off or unavailable should also be white, as with the original light cards. Instead they are currently transparent

If I find the time, I will look into a fix myself. But I thought I'd record it here already

**Comment by basbruss** [comment]
Thanks for the mention. I am already working to improve the code for all types of light cards and adding the horizontal_slider cards. So I will also look to this bug.
Do you use the `card_light_icon_info` card? As I am only using the colored ones and they do work as expected.

**Comment by CM000n** [comment]
Thx, I'm using the `card_light_color` template

**Comment by basbruss** [comment]
> Thx, I'm using the `card_light_color` template

And do you use light/dark mode switching or only light mode?
I will look now into it.

**Comment by CM000n** [comment]
I personally use only the light theme. But with the dark theme it's the same issue for me:
![657657nt](https://user-images.githubusercontent.com/7945681/147230618-dff92bb9-6234-4e05-9cf5-1763536efd27.PNG)

**Comment by basbruss** [comment]
I've fixed it. The template `icon_info_light` has no background color and alpha so it can be used in all the different type of cards. Adding a background-color to all states fixed it as shown below. :)
```
card_light_color:
  template:
    - "icon_info_light"
    - "ulm_language_variables"
  variables:
    ulm_card_light_name: "[[[ return entity.attributes.friendly_name ]]]"
    ulm_card_light_icon:
  show_icon: false
  show_name: false
  show_label: false
  state:
    - operator: "template"
      value: "[[[ return entity.state == 'on' ]]]"
      styles:
        card:
          - background-color: >
              [[[
                if (states['sun.sun'].state == "below_horizon"){
                  var color = entity.attributes.rgb_color;
                  if (color){
                     return 'rgba(' + color + ',0.1)'
                  }
                  else{
                    return 'rgba(var(--color-yellow),0.1)'
                  }
                }else if (states['sun.sun'].state == "above_horizon")
                  return 'rgba(var(--color-background-yellow),var(--opacity-bg))';
              ]]]
  styles:
    card:
      - border-radius: "var(--border-radius)"
      - box-shadow: "var(--box-shadow)"
      - padding: "12px"
      - background-color: "rgba(var(--color-background-yellow),var(--opacity-bg))" ## fixed by adding this
```

**Comment by basbruss** [comment]
I've fixed the problem. This card takes the background values from the `icon_info_light` template. In this template the background is set to no color and alpha to use in the other light cards. I've now provide a solutions which overrides this and use the standard card background color of the theme as shown below: 
```
card_light_color:
  template:
    - "icon_info_light"
    - "ulm_language_variables"
  variables:
    ulm_card_light_name: "[[[ return entity.attributes.friendly_name ]]]"
    ulm_card_light_icon:
  show_icon: false
  show_name: false
  show_label: false
  state:
    - operator: "template"
      value: "[[[ return entity.state == 'on' ]]]"
      styles:
        card:
          - background-color: >
              [[[
                if (states['sun.sun'].state == "below_horizon"){
                  var color = entity.attributes.rgb_color;
                  if (color){
                     return 'rgba(' + color + ',0.1)'
                  }
                  else{
                    return 'rgba(var(--color-yellow),0.1)'
                  }
                }else if (states['sun.sun'].state == "above_horizon")
                  return 'rgba(var(--color-background-yellow),var(--opacity-bg))';
              ]]]
  styles:
    card:
      - border-radius: "var(--border-radius)"
      - box-shadow: "var(--box-shadow)"
      - padding: "12px"
      ##by adding this it overrides the template and uses the theme standard.
      - background-color: 
```

**Comment by basbruss** [comment]
Fixed in #140
